app/src/modules/settings/routes/policies/collection.vue

```diff
@@ -0,0 +1,277 @@
+<script setup lang="ts">
+import { Header as TableHeader } from '@/components/v-table/types';
+import { fetchAll } from '@/utils/fetch-all';
+import { translate } from '@/utils/translate-object-values';
+import { unexpectedError } from '@/utils/unexpected-error';
+import SearchInput from '@/views/private/components/search-input.vue';
+import { Policy } from '@directus/types';
+import { computed, ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { useRouter } from 'vue-router';
+import SettingsNavigation from '../../components/navigation.vue';
+
+// Taken from 20240328A-permissions-policies.ts
+const PUBLIC_POLICY_ID = 'abf8a154-5b1c-4a46-ac9c-7300570f4f17';
```

Comment:
We should move / pull this from a single shared constant in `@directus/constants`